tasks/test_syntax.js

```diff
@@ -188,6 +190,47 @@ function assertCircularDeps() {
     });
 }
 
+// Ensure no ES6 has snuck through into the build:
+function assertES5() {
+    var CLIEngine = eslint.CLIEngine;
+
+    var cli = new CLIEngine({
+        useEslintrc: false,
+        ignore: false,
+        parserOptions: {
+            ecmaVersion: 5
+        }
+    });
+
+    // Filter out min and plotly-geo-assets.js since one is unnecessary
+    // and the other is super slow:
+    var files = fs.readdirSync(path.join(__dirname, '../dist'));
+    var validFiles = [];
+    for(var i = 0; i < files.length; i++) {
+        var f = files[i];
+        var isMin = !/[^(min)]\.js$/.test(f);
```

Comment:
Fixed. Included `plotly.js` but left out with `with-meta` to save a few seconds.